python/tank/deploy/util.py

```diff
@@ -38,6 +38,39 @@ def execute_git_command(cmd):
     if status != 0:
         raise TankError("Error executing git operation. The git command '%s' returned error code %s." % (cmd, status))     
 
+def execute_toolkit_command(pipeline_config_path, toolkit_command, args):
```

Comment:
oh i see. Yes, i think it's better to call it `execute_tank_command`. I think that describes better what it does! It's should probably be the other way around, that we in an ideal world should rename the *other* method to `execute_tank_command` :) But for now, let's worry about this one since it's part of the core API :)